recipe/meta.yaml

```diff
@@ -27,7 +27,8 @@ requirements:
   run:
     - python >=3.7
     - python
-    - conda-package-streaming >=0.7.0
+    - conda-package-streaming >=0.9.0
+    - zstandard
```

Comment:
Ours: https://github.com/AnacondaRecipes/conda-package-handling-feedstock/pull/10/files

Reply:
ah great! will add the constraint on zstandard